README.md

```diff
@@ -89,11 +74,25 @@ For general use, use the `main` branch along with [version 2.15.4 of the CodeQL
     
     CodeQL's analysis output is provided in the form of a SARIF log file. For a human readable format, drop the SARIF file into [SARIF Viewer Website](https://microsoft.github.io/sarif-web-component/). (If there are violations, they will show up. If not, the page will not update.)
     ```
-    D:\codeql-home\codeql>codeql database analyze <path to database> --format=sarifv2.1.0 --output=<"path to output file".sarif> <path to query/suite to run>
+    codeql database analyze <path to database> microsoft/windows-drivers --format=sarifv2.1.0 --output=<"path to output file".sarif> 
+    ```
+    By default, the `microsoft/windows-drivers` CodeQL pack will run the windows_driver_mustfix query suite. However, specific versions, queries, or suites can be specified using the format `codeql database analyze <database> <scope>/<pack>@x.x.x:<path>`. For futher information, see the [CodeQL documentation](https://docs.github.com/en/code-security/codeql-cli/using-the-advanced-functionality-of-the-codeql-cli/publishing-and-using-codeql-packs#using-a-codeql-pack-to-analyze-a-codeql-database)
```

Comment:
This is less a documentation point and more a policy one - do you think it'd be better to have the pack run recommended by default, since that's a superset of must-fix? That way users get more queries running and we get more feedback.  We can include a note that the must-fix suite is also available.